scripts/check_droid_tag_workflow.py

```
@@ -0,0 +1,187 @@
+    errors: list[str] = []
+
+    on_match = re.search(r"(?ms)^on:\n(?P<body>.*?)(?=^[A-Za-z0-9_-]+:\n)", text)
+    if not on_match:
+        errors.append("missing top-level on block")
+        on_block = ""
+    else:
+        on_block = on_match.group("body")
+
+    for event in ("issue_comment", "pull_request_review_comment", "pull_request_review"):
+        if not re.search(rf"(?m)^  {event}:\s*$", on_block):
+            errors.append(f"missing allowed trigger: {event}")
```

Ruff (0.16.3)

[warning] Use `list.extend` to create a transformed list

(PERF401)